httpie/output/formatters/xml.py

```diff
@@ -26,9 +40,18 @@ def pretty_xml(document: 'Document',
     if standalone is not None and sys.version_info >= (3, 9):
         kwargs['standalone'] = standalone
     body = document.toprettyxml(**kwargs).decode(kwargs['encoding'])
+    lines = [line for line in body.splitlines() if line.strip()]
+
+    # Remove the declaration
+    if parse_declaration(lines[0]):
```

Comment:
Will there always be `lines[0]`?

Reply:
> Will there always be lines[0]?

That is a good question, and from my understanding the answer is yes. Everytime we convert the document to a string, the root of it will provide unconditional declaration (https://github.com/python/cpython/blob/851811f5772c43f72f445e2ce1ac3ea9da951ae3/Lib/xml/dom/minidom.py#L1823).